codedeploy_role_document.json

```diff
@@ -0,0 +1,17 @@
+{
+  "Version": "2012-10-17",
+  "Statement": [{
+    "Sid": "",
+    "Effect": "Allow",
+    "Principal": {
+      "Service": [
+        "codedeploy.us-east-1.amazonaws.com",
+        "codedeploy.eu-west-1.amazonaws.com",
+        "codedeploy.us-west-2.amazonaws.com",
```

Comment:
yes, they're in `eu-west-1`.  They're not in `us-east-1` or `us-west-2`, so I'm not sure why these lines are required.